deployments/apiv2/services/users/src/main.py

```diff
@@ -327,3 +340,78 @@ async def register(
     except Exception as e:
         logger.exception(f"register: Unexpected error {repr(e)}")
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@app.get("/users")
+async def get_users(request: Request, token=Depends(ProtectedAny(scope=["users:admin"]))):
+    """Get all the information for all the users connected to the admin account that started this request.
+    By default it will be sorted with all active users showing up first, then all the deactivated users
```

Comment:
Is there any reason why we need to sort the users by their suspended status in this route? If sorting is needed in the FE, we could always handle it there